internal/pkg/v2/ui/http/acceptance/common/ping/v2dot0/cases.go

```diff
@@ -0,0 +1,294 @@
+/*******************************************************************************
+ * Copyright 2020 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package v2dot0
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/application"
+	dtoBaseV2dot0 "github.com/edgexfoundry/edgex-go/internal/pkg/v2/application/dto/v2dot0/common/base"
+	dtoErrorV2dot0 "github.com/edgexfoundry/edgex-go/internal/pkg/v2/application/dto/v2dot0/common/error"
+	dtoV2dot0 "github.com/edgexfoundry/edgex-go/internal/pkg/v2/application/dto/v2dot0/common/ping"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/infrastructure/test"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/ui/common/batchdto"
+)
+
+// UseCaseTestCases returns a series of v2.0 test cases to test the ping use-case endpoint.
+func UseCaseTestCases(t *testing.T) []*test.Case {
+	return []*test.Case{
```

Comment:
This func could use some additional comments...

Reply:
I'm not sure what's not clear here so I'm not sure what additional comments are required.  Please provide some additional details @tonyespy.